src/i18n-scanner.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const glob = require('glob');
+const Parser = require('i18next-scanner').Parser;
+
+const translationFunctionNames = ['i18next.t', 'props.t', 'this.props.t', 't'];
+const outputFilePath = './src/locales/en/common.json';
```

Comment:
Are we supposed to update the json file of other languages manually? I don't mean the values, but the keys.

Reply:
Other translation files are supposed to be changed by the localization platform set up in https://github.com/LiskHQ/lisk-nano/issues/736

In the end, I'd prefer to .gitignore the translation files and fetching them from the localization platform on a build.